common-custom-user-data-scripts/maven-common-custom-user-data.groovy

```diff
@@ -37,7 +37,7 @@ static void tagOs(def buildScan) {
 }
 
```

Comment:
I will soon delete the Groovy file for Maven. But, can you please also update the custom data script for Gradle? Thanks.

Reply:
Gradle is not affected by this issue since the system property is set correctly in the from `-Didea.version=<version>` for Gradle invocations. At least for IntelliJ 2020.1 and 2020.2. I haven't checked older IntelliJ releases.

Reply:
My bad. Sorry about that.